kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -740,6 +740,19 @@
   .main-content-grid {
     position: relative;
     top: $toolbar-height;
+
+    /*
+      Fixes jumping scrollbar when reaching the bottom of the page
+      for certain page heights. The issue is caused by the document
+      scroll height being changed by the sidebar's switching position
+      from absolute to fixed in the sticky calculation, resulting in
+      an endless cycle of the sticky calculation being called
+      and the sidepanel alternating between fixed and absolute position
+      over and over. Setting min width prevents this by making sure that
```

Comment:
Should this be `min height` not `min width`?

Reply:
Also - does this happen across all breakpoints? Just thinking about the blanket application of this on mobile.

Reply:
Thank you, yes it should say "min height", will fix it.

> does this happen across all breakpoints? Just thinking about the blanket application of this on mobile.

That's a good point, I don't know. I assumed that in the blanket state, there would be a sidebar anyways, but I really didn't test it and it may not be the case in all cases because `TopicsPage` has many modes. Adding this style just for some breakpoints may be safer. I will look at it.